src/components/task-plan/builder/date-time.cjsx

```diff
@@ -80,7 +80,7 @@ DateTime = React.createClass
     @state?.date? and _.isEmpty(@refs?.date?.state?.errors)
 
   isTimeValid: ->
-    @state?.time? and _.isEmpty @refs?.time?.refs?.timeInput?.state?.errors
+    @state?.time? and _.isEmpty(@refs?.time?.refs?.timeInput?.state?.errors)
 
```

Comment:
Hey, I just looked at this a bit closer and noticed that `@ref.time` is set to the `TutorTimeInput` which looks like it will always render.  What do you think about adding a `isEmpty` method to  `TutorTimeInput`.  Then this could be just `@state?.time? and @refs.time.isEmpty()`